### PR TITLE
[FEAT] Message component

### PR DESCRIPTION
**Pull request by yv2003**

### CODE REVIEW

File: src/components/About/Message/index.astro [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+<link rel="preconnect" href="https://fonts.googleapis.com" />
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+<link
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+    rel="stylesheet"
+/>
+<message class="inline-block px-5 py-4 text-justify">
+<!-- <message class="main"> -->
+    <div class="pt-2.5 pr-2.5 pl-2.5  font-normal text-4xl leading-10 gap-10 tracking-tight" style="font-family: Fraunces">{title}</div>
```
Comment (JayeshVP24) [inline src/components/About/Message/index.astro:14]: remove style="font-famil....
add tailwind class `font-title`
font-title is fraunces

File: src/components/About/Message/index.astro [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <div class="pt-3 gap-4">
+        <div class="flex mx-auto w-72 h-120 pt-5 pr-5 pb-5 md:float-left">
+            <img class="rounded-lg" src={img} />
+        </div>
+        <div class="pt-3 p-1">
+            <p
+                class="font-[Inter] text-base font-normal text-[#475467] first-line:tracking-widest first-letter:text-4xl first-letter:font-bold first-letter:text-[#475467] first-letter:mr-1 first-letter:float-leftmd:text-justify">
```
Comment (JayeshVP24) [inline src/components/About/Message/index.astro:21]: remove font Inter, remove first line tracking widest, remove first letter text -[#... , remove first letter float

File: src/components/About/Message/index.astro [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                {message}
+            </p>
+            <p class="pt-2 font-[Inter] text-xl font-[600] text-[#475467] italic">
```
Comment (JayeshVP24) [inline src/components/About/Message/index.astro:24]: remove font inter
inter is default

Comment (JayeshVP24) [reply src/components/About/Message/index.astro:24]: remove italic, check figma properly. Our aim is to make it as close to figma as possible

File: src/components/About/Message/index.astro [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                {name}
+            </p>
+        </div>
+    </div>
+</message>
+<style>
```
Comment (JayeshVP24) [inline src/components/About/Message/index.astro:30]: delete all style tag full

File: src/components/About/Message/index.astro [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
+const { title, img, message, name } = Astro.props;
+// use localhost:3000/about/pages to access all the links
+---
+
+<link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (JayeshVP24) [inline src/components/About/Message/index.astro:6]: remove all this links of fonts. I have added font in layout already

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]
Also check the notion file pinned in frontend chat. wait
https://tcet-opensource.notion.site/TCET-Mumbai-Docs-f92526652a9a4facbc4b74d0ffe1a612?pvs=4
here, 
the file names of all message pages should be same as the href in this

File: src/components/About/Message/index.astro [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+const { title, img, message, name } = Astro.props;
+// use localhost:3000/page to access all the links
+---
+
+
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+<link
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+    rel="stylesheet"
+/>
```
Comment (JayeshVP24) [inline src/components/About/Message/index.astro:11]: remove these lines

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

**Review by kamalika0363 — APPROVED** [review]
LGTM

**Pull request merged**